buf/registry/owner/v1/user_service.proto

```diff
@@ -97,12 +97,14 @@ message ListUsersRequest {
   // Only return Users of these types.
   repeated UserType has_types = 5 [
     (buf.validate.field).repeated.unique = true,
-    (buf.validate.field).repeated.items.enum.defined_only = true
+    (buf.validate.field).repeated.items.enum.defined_only = true,
+    (buf.validate.field).repeated.items.enum.not_in = 0
```

Comment:
Question for this: when `UNSPECIFIED` is passed, we are currently falling back to the same type as `STANDARD`, however that functionally bypasses the uniqueness check (e.g. if they have both `STANDARD` and `UNSPECIFIED`). Should we continue disallow this?

Reply:
For a repeated field of enums that's intended to be unique as well, it makes sense to me to exclude UNSPECIFIED as a valid option. In this case, a user cannot really have an UNSPECIFIED type.

Reply:
Excellent, I'll keep this restriction in that case. Thank you!